ovn_cluster.sh

```diff
@@ -438,26 +500,67 @@ function start-db-cluster() {
     sleep 3
 
     # Start ovn-northd on all ovn-central nodes. One of the instance gets the lock from
-    # SB DB ovsdb-server and becomes active. Most likely ovn-northd in ${CENTRAL_NAME}-1
+    # SB DB ovsdb-server and becomes active. Most likely ovn-northd in ${name}-1
     # will become active as it is started first.
     # 'ovn-appctl -t ovn-northd status' will give the status of ovn-northd i.e if it
     # has lock and active or not.
-    ${RUNC_CMD} exec ${CENTRAL_NAME}-1 ${OVNCTL_PATH}  \
+    ${RUNC_CMD} exec ${name}-1 ${OVNCTL_PATH}  \
     --ovn-northd-nb-db=${REMOTE_PROT}:${central_1_ip}:6641,${REMOTE_PROT}:${central_2_ip}:6641,${REMOTE_PROT}:${central_3_ip}:6641 \
     --ovn-northd-sb-db=${REMOTE_PROT}:${central_1_ip}:6642,${REMOTE_PROT}:${central_2_ip}:6642,${REMOTE_PROT}:${central_3_ip}:6642 --ovn-manage-ovsdb=no \
     $SSL_ARGS start_northd
 
-    ${RUNC_CMD} exec ${CENTRAL_NAME}-2 ${OVNCTL_PATH}  \
+    ${RUNC_CMD} exec ${name}-2 ${OVNCTL_PATH}  \
     --ovn-northd-nb-db=${REMOTE_PROT}:${central_1_ip}:6641,${REMOTE_PROT}:${central_2_ip}:6641,${REMOTE_PROT}:${central_3_ip}:6641 \
     --ovn-northd-sb-db=${REMOTE_PROT}:${central_1_ip}:6642,${REMOTE_PROT}:${central_2_ip}:6642,${REMOTE_PROT}:${central_3_ip}:6642 --ovn-manage-ovsdb=no \
     $SSL_ARGS start_northd
 
-    ${RUNC_CMD} exec ${CENTRAL_NAME}-3 ${OVNCTL_PATH}  \
+    ${RUNC_CMD} exec ${name}-3 ${OVNCTL_PATH}  \
     --ovn-northd-nb-db=${REMOTE_PROT}:${central_1_ip}:6641,${REMOTE_PROT}:${central_2_ip}:6641,${REMOTE_PROT}:${central_3_ip}:6641 \
     --ovn-northd-sb-db=${REMOTE_PROT}:${central_1_ip}:6642,${REMOTE_PROT}:${central_2_ip}:6642,${REMOTE_PROT}:${central_3_ip}:6642 --ovn-manage-ovsdb=no \
     $SSL_ARGS start_northd
 }
 
+function start-ovn-ic() {
+    if [ "$OVN_DB_CLUSTER" = "yes" ]; then
+        central_ic=$(awk -v NAME="${CENTRAL_NAMES[0]}" 'match($0, NAME) {print $2}' _ovn_central_1)
+        central_ic_id="${CENTRAL_NAMES[0]}"-1
+        ${RUNC_CMD} exec ${central_ic_id} ${OVNCTL_PATH}  \
+            --db-ic-nb-create-insecure-remote=yes   \
+            --db-ic-sb-create-insecure-remote=yes start_ic_ovsdb
+    else
+        [ $RELAY_COUNT -gt 0 ] && REMOTE=_ovn_remote_main_db || REMOTE=_ovn_remote
+        central_ic=$(awk -v NAME="${CENTRAL_NAMES[0]}" 'match($0, NAME) {print $2}' $REMOTE |awk -F: '{print $2}')
+        central_ic_id="${CENTRAL_NAMES[0]}"
+        ${RUNC_CMD} exec ${central_ic_id} ${OVNCTL_PATH}  \
+            --db-ic-nb-create-insecure-remote=yes   \
+            --db-ic-sb-create-insecure-remote=yes start_ic_ovsdb
+    fi
```

Comment:
I think it would be useful if we added a new environment variable, OVN_START_IC_DBS (default "yes"), and move all the code that starts the IC DBs under if [ "$OVN_START_IC_DBS" = "yes" ]; then

It would also be beneficial if a user could specify the IC DBs node IP and ID when that variable is not set.

What I had in mind is something like:
```
    if [ "$OVN_START_IC_DBS" = "yes" ]; then
        if [ "$OVN_DB_CLUSTER" = "yes" ]; then
            central_ic=$(awk -v NAME="${CENTRAL_NAMES[0]}" 'match($0, NAME) {print $2}' _ovn_central_1)
            central_ic_id="${CENTRAL_NAMES[0]}"-1
            ${RUNC_CMD} exec ${central_ic_id} ${OVNCTL_PATH}  \
                --db-ic-nb-create-insecure-remote=yes   \
                --db-ic-sb-create-insecure-remote=yes start_ic_ovsdb
        else
            central_ic=$(awk -v NAME="${CENTRAL_NAMES[0]}" 'match($0, NAME) {print $2}' _ovn_remote |awk -F: '{print $2}')
            central_ic_id="${CENTRAL_NAMES[0]}"
            ${RUNC_CMD} exec ${central_ic_id} ${OVNCTL_PATH}  \
                --db-ic-nb-create-insecure-remote=yes   \
                --db-ic-sb-create-insecure-remote=yes start_ic_ovsdb
        fi
    elif [ -z "$CENTRAL_IC_IP" ] || [ -z "$CENTRAL_IC_ID" ]; then
        echo "IC DBs not started locally, please specify the IC DBs container name (CENTRAL_IC_ID) and its IP (CENTRAL_IC_IP)"
        exit 1
    else
        central_ic=$CENTRAL_IC_IP
        central_ic_id=$CENTRAL_IC_ID
    fi
    ```

Reply:
@dceara ack, I will fix it